source/grid/grid_tools.inst.in

```diff
@@ -133,6 +133,14 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension :  SPACE_DIMENSIONS
       void scale<deal_II_dimension> (const double,
                                      Triangulation<deal_II_dimension, deal_II_space_dimension> &);
 
+#if deal_II_space_dimension==3
+    template
+      void
+      rotate<deal_II_dimension> (const double,
+                                 const unsigned int ,
+                                 Triangulation<deal_II_dimension, deal_II_space_dimension> &);
```

Comment:
> which means that the instantiation only happens for dim==3, no?

No, I am instantiating for dim=1,2,3 and spacedim=3. At least I hope that is what happens.